bin/tasks-header.md

```diff
@@ -0,0 +1,6 @@
+# Cumulus Tasks
+
+A list of reusable Cumulus tasks. [Add your own.](docs/adding-a-task.md)
```

Comment:
This link does not work. @kkelly51 fixed this issue with other links but I forget how and would have to dig.